cmake/defaults/Options.cmake

```diff
@@ -47,6 +47,7 @@ option(PXR_ENABLE_OSL_SUPPORT "Enable OSL (OpenShadingLanguage) based components
 option(PXR_ENABLE_PTEX_SUPPORT "Enable Ptex support" ON)
 option(PXR_ENABLE_OPENVDB_SUPPORT "Enable OpenVDB support" OFF)
 option(PXR_ENABLE_NAMESPACES "Enable C++ namespaces." ON)
+option(PXR_DEFINE_BOOST_DEBUG_PYTHON_FLAG "Define BOOST_DEBUG_PYTHON flag." OFF)
```

Comment:
It looks like this flag controls stuff beyond just setting the necessary boost defines, for example appending "_d" to library names on Windows. If we ever switched to pybind11 or something, we'd drop the boost defines but would still need to leave the library name behavior in place.

Because of that, it seems like this flag should be named more generally, like `PXR_USE_DEBUG_PYTHON`. This would match the `--debug-python` option in build_usd.py. We could also move this flag next to the other Python-related options at line 43-44.

Reply:
Right the "_d" is a python thing, not a boost thing. I'm good with renaming this option and moving it with the other python ones.